### PR TITLE
Various updates of SMALL

### PR DESCRIPTION
**Pull request by josla972**
Mostly changes taken from xSMALL to be up to date with newer FreeBSD src.

### CODE REVIEW

File: profiles/SMALL_/profile.mk [hunk 1]
```diff
@@ -1,27 +1,38 @@
 
 WORLD_SUBDIRS_LIB+= \
 libalias \
+lib80211 \
```
Comment (rayddteam) [inline profiles/SMALL_/profile.mk]: lib80211 should be part of wireless profile. Not all routers have wireless.

File: profiles/SMALL_/profile.mk [hunk 2]
```diff
@@ -1,27 +1,38 @@
...
+libmd \
 libpmc \
 libsbuf \
 libtacplus \
 libthr \
 libufs \
 libutil \
+libwrap \
+libxo \
+libwrap \
```
Comment (rayddteam) [inline profiles/SMALL_/profile.mk]: dup

File: profiles/SMALL_/profile.mk (hunk 1 above)
Comment (yamori813) [reply profiles/SMALL_/profile.mk]: Current source ifconfig command depend lib80211...

Comment (rayddteam) [reply profiles/SMALL_/profile.mk]: Ahh, indeed. sorry for noise

File: profiles/SMALL_/profile.mk (hunk 2 above)
Comment (josla972) [reply profiles/SMALL_/profile.mk]: Ok, will fix later today!

File: profiles/SMALL_/profile.mk (hunk 1 above)
Comment (yamori813) [reply profiles/SMALL_/profile.mk]: This is my check lib dependency script by ruby.  Execute on ZRouter obj directory.

https://gist.github.com/yamori813/b6bc36f6e7373cc231ac0eb5038d1496

**Comment by rayddteam** [comment]
Yup
Thanks

**Pull request merged**